packages/terra-show-hide/src/_ShowHideButton.module.scss

```diff
@@ -0,0 +1,103 @@
+    line-height: var(--terra-show-hide-button-line-height, 1.85714);
+    outline: none;
+    padding: 0;
+    text-align: center;
+    text-decoration: var(--terra-show-hide-button-underline, none);
+    text-transform: none;
+    touch-action: manipulation; // Enable fast tap interaction in webkit browsers; see https://webkit.org/blog/5610/more-responsive-tapping-on-ios/
+    user-select: none; // Prevent text selection on buttons on mobile devices
+    white-space: normal;
+    // Remove the inner border and padding in Firefox.
+    &::-moz-focus-inner {
+      border: 0;
+      padding: 0;
+    }
+
+    &::after {
+      background-image: var(--terra-show-hide-button-icon, inline-svg('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><polygon fill="#0059a8" points="40.8,8.6 24,25 7.2,8.6 0,15.9 24,39.4 48,15.9 "/></svg>'));
+      background-repeat: no-repeat;
+      background-size: cover;
+      content: '';
+      display: inline-block;
+      height: var(--terra-show-hide-button-icon-height, 0.7143em);
+      margin-left: var(--terra-show-hide-button-icon-margin-left, 0.28571em);
+      width: var(--terra-show-hide-button-icon-width, 0.7143em);
+    }
+
+    &.is-focused {
+      background-color: var(--terra-show-hide-button-focus-background-color, transparent);
+      border-radius: var(--terra-show-hide-button-focus-border-radius, 0.2142rem);
+      box-shadow: var(--terra-show-hide-button-focus-box-shadow, 0 0 0 1px #4cb2e9, 0 0 0 3px rgba(76, 178, 233, 0.25));
+      color: var(--terra-show-hide-button-focus-color, #001f67);
+      text-decoration: var(--terra-show-hide-button-focus-underline, underline);
+
+      &::after {
+        background-image: var(--terra-show-hide-button-focus-icon, inline-svg('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><polygon fill="#0059a8" points="40.8,8.6 24,25 7.2,8.6 0,15.9 24,39.4 48,15.9 "/></svg>'));
+      }
+    }
+
+    &:hover,
+    &.is-hovered {
+      background-color: var(--terra-show-hide-button-hover-background-color);
+      color: var(--terra-show-hide-button-hover-color, #001f67);
+      text-decoration: var(--terra-show-hide-button-hover-text-decoration, underline);
+
+      &::after {
+        background-image: var(--terra-show-hide-button-hover-icon, inline-svg('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><polygon fill="#0059a8" points="40.8,8.6 24,25 7.2,8.6 0,15.9 24,39.4 48,15.9 "/></svg>'));
+      }
+    }
+
+    &:active {
+      color: var(--terra-show-hide-button-active-color, #001f67);
+    }
+
+    &:disabled,
+    &.is-disabled {
+      color: var(--terra-show-hide-button-disabled-color, rgba(28, 31, 33, 0.2));
+      pointer-events: none;
+      text-decoration: none;
+
+      &::after {
+        background-image: var(--terra-show-hide-button-disabled-icon, inline-svg('svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><polygon fill="#0059a8" points="40.8,8.6 24,25 7.2,8.6 0,15.9 24,39.4 48,15.9 "/></svg>'));
```

Comment:
```suggestion
        background-image: var(--terra-show-hide-button-disabled-icon, inline-svg('svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><polygon fill="rgba(28, 31, 33, 0.2)" points="40.8,8.6 24,25 7.2,8.6 0,15.9 24,39.4 48,15.9 "/></svg>'));
```

Reply:
(fyi - after updating - you will need to regenerate/update the screenshots for the 3 states: hover,focus,inactive)